test/pummel/test-buffer-large-size-buffer-alloc-unsafe-slow.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+const common = require('../common');
+
+// Buffer with size > INT32_MAX
+common.skipIf32Bits();
+
+// Test Buffer size larger than integer range
+const { test } = require('node:test');
+const assert = require('assert');
+const kStringMaxLength = require('buffer').constants.MAX_STRING_LENGTH;
+
+const stringTooLongError = {
+  message: `Cannot create a string longer than 0x${kStringMaxLength.toString(16)}` +
+    ' characters',
+  code: 'ERR_STRING_TOO_LONG',
+  name: 'Error',
+};
```

Comment:
This can be simplified by just testing if it throws an error with code `ERR_STRING_TOO_LONG` instead of testing the error message

Reply:
makes sense, thanks!